internal/gatewayapi/testdata/httproute-with-multiple-gateways-from-different-ns.out.yaml

```diff
@@ -193,8 +196,16 @@ xdsIR:
         mergeSlashes: true
       port: 10080
       routes:
-      - directResponse:
```

Comment:
doesnt look right

Reply:
This test should translate the route without `directResponse` because both the 2 Gateways allow routes from different ns, and the hostname matches.

Reply:
This change is actually irrelevant to the status:  I found that `httproute-with-multiple-gateways-from-different-ns.in.yaml` is invalid, missing the backend, so I added it.

The purpose of this test is to verify we have the right namespace for the parent Gateway in the route status, see: https://github.com/envoyproxy/gateway/pull/4592